acceptance/shared/libraries/kitchen.rb

```diff
@@ -0,0 +1,30 @@
+class Kitchen < Chef::Resource
```

Comment:
I built something very similar.  I would like to move this to chef-acceptance so all projects benefit from it.